conf/nginx/st2.conf

```diff
@@ -110,6 +116,7 @@ server {
     error_page 502 = @authError;
 
     rewrite ^/auth/(.*)  /$1 break;
+    server_tokens off;
```

Comment:
Sorry if I wasn't clear enough in my previous comment.

`server_tokens off;` would be sufficient for the `server` blocks and not needed to repeat for every possible `location` as they'll inherit it. This way we'll also avoid duplication.

Can you try that?

Reply:
Thanks for the changes @ashwini-orchestral  :+1: